rust/hyperlane-base/src/settings/loader.rs

```diff
@@ -44,16 +47,57 @@ pub(crate) fn load_settings_object<'de, T: Deserialize<'de>, S: AsRef<str>>(
     let builder = Config::builder();
 
     // Load the base config file the old way
-    let builder = match (env::var("RUN_ENV").ok(), env::var("BASE_CONFIG").ok()) {
-        (Some(env), Some(fname)) => {
-            builder.add_source(File::with_name(&format!("./config/{}/{}", env, fname)))
+    let builder = match (env::var("RUN_ENV"), env::var("BASE_CONFIG")) {
+        (Ok(env), Ok(fname)) => {
+            builder.add_source(File::with_name(&format!("./config/{env}/{fname}")))
+        }
+        (Ok(env), _) => {
```

Comment:
Not sure if this case adds much value. I also might be able to dedup some of this code but was having issues because the `builder` uses static typing in a way that was making things difficult.

Reply:
Why not remove RUN_ENV and BASE_CONFIG entirely?

Reply:
Done